service/version_bundles.go

```diff
@@ -313,8 +313,8 @@ func NewVersionBundles() []versionbundle.Bundle {
 		{
 			Changelogs: []versionbundle.Changelog{
 				{
-					Component:   "cloudformation",
-					Description: "Add missing region to host account credentials.",
+					Component:   "cloudconfig",
+					Description: "Add OIDC integration for k8s api-server.",
```

Comment:
Nitpick: `k8s` should be `Kubernetes`. 

The changelogs are exposed to customers via the UI.